src/src_V2/modules/manager/financial/payables/control-panel/ControlPanel.jsx

```diff
@@ -0,0 +1,31 @@
+import { FilterAlt } from '@mui/icons-material';
+import { Box, Button } from '@mui/material';
+
+import { SearchInput } from '../../components';
+import styles from './ControlPanel.module.css';
+
+export default function ControlPanel({ searchValue, handleSearchChange }) {
+  return (
+    <Box className={styles.mainDiv}>
+      <Box className={styles.search}>
+        <SearchInput
+          value={searchValue}
+          label='Search by Tour Name/ID/Supplier'
+          searchHandler={handleSearchChange}
+        />
+      </Box>
+
+      <Box className={styles.filterDiv}>
+        <Button variant='outlined' startIcon={<FilterAlt />}>
+          Filter
+        </Button>
+      </Box>
+
+      {/*<Box className={styles.optionsDiv}>*/}
```

Comment:
@aninaslyan jan need we this commented section?

Reply:
Maybe we'll manage or in the future, someone else manages to implement the functionality of the button. The button also has corresponding styles. That's why I left it here.